lib/localPeers.js

```diff
@@ -106,7 +105,6 @@ api.getKeyPair = async ({ledgerNodeId}) => {
     _pruneCacheTimer = setTimeout(
       _pruneCache, LOCAL_PEER_KEY_PAIR_CACHE_MAX_AGE);
   }
-
```

Comment:
Let's keep this whitespace.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/b18261564956b343ca5b00925f3d5c551aa482b1